cli/api/daemon.go

```diff
@@ -625,6 +625,34 @@ func (d *daemon) startAgent() error {
 		"hostid": myHostID,
 	})
 
+	// Load delegate keys if they exist
+	delegateKeyFile := path.Join(options.IsvcsPath, auth.DelegateKeyFileName)
+	keylog := log.WithFields(logrus.Fields{
+		"keyfile": delegateKeyFile,
+	})
+
+	go func() {
+		auth.WatchForDelegateKeys(delegateKeyFile, d.shutdown)
```

Comment:
Should we continue to watch for changes to the key file after it has been loaded the first time?  Otherwise if we need to reissue the key we will have to restart the delegate.